### PR TITLE
Finish Capybara tests in features/admin/withdraw_spec.rb

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Finished Capybara tests in features/admin/withdraw_spec.rb #831

### CODE REVIEW

File: spec/features/admin/withdraw_spec.rb [hunk 1]
```diff
@@ -2,71 +2,107 @@
   let!(:member) { create :member, :verified_identity }
   let!(:admin_member) { create :member, :verified_identity, email: Member.admins.first }
 
-  let!(:account) do
+  let!(:usd_account) do
     member.get_account(:usd).tap { |a| a.update_attributes locked: 8000, balance: 10_000 }
```
Comment (yivo) [inline spec/features/admin/withdraw_spec.rb]: @dinesh-skyach Can you use `update!` and round braces `()` here and in other places?

File: spec/features/admin/withdraw_spec.rb [hunk 2]
```diff
@@ -2,71 +2,107 @@
...
+    within '#dashboard-index' do
       expect(page).to have_content(I18n.t('layouts.admin.menus.items.operating.withdraws'))
       click_on I18n.t('layouts.admin.menus.items.operating.withdraws')
     end
   end
 
-  it 'admin view withdraws' do
-    pending 'skip withdraw dashboard'
 
+  it 'admin view usd withdraws' do
```
Comment (yivo) [inline spec/features/admin/withdraw_spec.rb]: @dinesh-skyach please use different way of writing messages. For example (the current) case: `it 'allows admin to view USD withdraws'`. Please, update other messages too.

File: spec/features/admin/withdraw_spec.rb [hunk 3]
```diff
@@ -2,71 +2,107 @@
...
-    expect(page).to have_content(withdraw.label)
-    expect(page).to have_content(I18n.t('actions.transact'))
-    expect(page).to have_content(I18n.t('actions.reject'))
+    click_link usd_withdraw.currency.code.upcase
+
+    expect(page).to have_content(usd_withdraw.rid)
+    expect(page).to have_content(usd_withdraw.amount)
+    click_link I18n.t('actions.view')
+    page.within_window(windows.last) do
+      expect(page).to have_content(I18n.t('actions.process'))
```
Comment (yivo) [inline spec/features/admin/withdraw_spec.rb]: @dinesh-skyach You can feel free to omit I18n for admin panel. In admin we support only English.

**Review by yivo — CHANGES_REQUESTED** [review]

File: spec/features/admin/withdraw_spec.rb (hunk 1 above)
Comment (dinesh-skyach) [reply spec/features/admin/withdraw_spec.rb]: @yivo changes done

**Review by yivo — APPROVED** [review]

**Pull request merged**